frontend/public/style/_overrides.scss

```diff
@@ -397,6 +397,10 @@ kbd {
   vertical-align: top;
 }
 
+.pf-c-table.pf-m-compact tr > td {
```

Comment:
Should we generalize the `.pf-c-table.pf-c-virtualized` rule above and update the comment instead?

At one point, I believe this caused problems for mobile display with table `gridBreakPoint`. We're not using that currently, however, so maybe it's a non-issue.

cc @rhamilto

Reply:
+1.  Since the declarations match, it'd be great to combine the rules in to a single one.

Reply:
```css
.pf-c-table.pf-m-compact,
// override .co-m-table-grid [class*='col-'], .co-m-table-grid .row
.pf-c-table.pf-c-virtualized {
  tr > td {
    vertical-align: top;
  }
}
```